NodeRequirer.sublime-settings

```diff
@@ -21,9 +21,11 @@
         "pouchdb": "PouchDB",
         "prova": "test",
         "es6-promise": "Promise",
+        "native-promise-only": "Promise",
         "ractive": "Ractive",
         "ramda": "R",
         "react": "React",
+        "roslib": "ROS",
```

Comment:
The only one I added is graceful-fs. This and all the others are mappings which were for some reason defined in `Require.sublime-settings` but not in `NodeRequirer.sublime-settings`.